src/darwin/CHIPTool/CHIPTool/View Controllers/RootViewController.m

```diff
@@ -43,7 +44,7 @@ - (void)setUpTableView
     [self.view addSubview:self.tableView];
     self.options = @[
         @"QRCode scanner", @"Echo client", @"Light on / off cluster", @"Temperature Sensor", @"Bindings", @"Wifi Configuration",
-        @"Enable Pairing", @"Unpair Devices", @"Fabric Management"
+        @"Enable Pairing", @"Unpair Devices", @"Fabric Management", @"Provision CHIP Device With Custom Flow"
```

Comment:
"Custom Commissioning Flow" would be a more appropriate terminology

Reply:
This part is removed since the logic is combined in QRCode scan